<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gui</class>
 <widget class="QMainWindow" name="Gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>900</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>gro 3.0: easy and fast agent-based models </string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionReload"/>
	<addaction name="actionReloadSeed"/>
    <addaction name="actionScreenshot"/>
   </widget>
   <widget class="QMenu" name="menuSimulation">
    <property name="title">
     <string>Simulation</string>
    </property>
    <addaction name="actionStartStop"/>
    <addaction name="actionStep"/>
	<addaction name="action1step"/>
	<addaction name="action10steps"/>
	<addaction name="action100steps"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
    <addaction name="actionCenter"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSimulation"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionReload"/>
   <addaction name="actionReloadSeed"/>
   <addaction name="separator"/>
   <addaction name="actionStartStop"/>
   <addaction name="actionStep"/>
   <addaction name="action1step"/>
   <addaction name="action10steps"/>
   <addaction name="action100steps"/>
   <addaction name="separator"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionAutoZoom"/>
   <addaction name="actionCenter"/>
   <addaction name="separator"/>
   <addaction name="actionShowPlots"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionStartStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icons/icons/start.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="toolTip">
    <string>Start or stop gro </string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionOpen">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icons/icons/open.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="iconText">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open a .gro file.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionReload">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icons/icons/reload.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
   <property name="toolTip">
    <string>Reload current .gro file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionReloadSeed">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icons/icons/reloadSeed.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Change seed</string>
   </property>
   <property name="toolTip">
    <string>Reload with a different seed</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionStep">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icons/icons/step.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Step to time</string>
   </property>
   <property name="toolTip">
    <string>Simulate until given time</string>
   </property>
   <property name="shortcut">
    <string>0</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Online documentation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionScreenshot">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Screenshot...</string>
   </property>
   <property name="toolTip">
    <string>Save screenshot</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icons/icons/zoom_in.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icons/icons/zoom_out.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="toolTip">
    <string>Zoom out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionCenter">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="center.png" resource="icons.qrc">
     <normaloff>:/icons/icons/center.png</normaloff>:/icons/icons/center.png</iconset>
   </property>
   <property name="text">
    <string>Center view</string>
   </property>
   <property name="toolTip">
    <string>Center view</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionAutoZoom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/zoom_auto.png</normaloff>:/icons/icons/zoom_auto.png</iconset>
   </property>
   <property name="text">
    <string>Auto Zoom</string>
   </property>
   <property name="toolTip">
    <string>Toggle auto zoom</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action10steps">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/x10.png</normaloff>:/icons/icons/x10.png</iconset>
   </property>
   <property name="text">
    <string>10 steps</string>
   </property>
   <property name="toolTip">
    <string>10 steps</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="action100steps">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/x100.png</normaloff>:/icons/icons/x100.png</iconset>
   </property>
   <property name="text">
    <string>100 steps</string>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
  </action>
  <action name="action1step">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/x1.png</normaloff>:/icons/icons/x1.png</iconset>
   </property>
   <property name="text">
    <string>1 step</string>
   </property>
   <property name="toolTip">
    <string>1 step</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
  </action>
  <action name="actionShowPlots">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/plots.png</normaloff>:/icons/icons/plots.png</iconset>
   </property>
   <property name="text">
    <string>ShowPlots</string>
   </property>
   <property name="toolTip">
    <string>show plots windows</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionStartStop</sender>
   <signal>triggered()</signal>
   <receiver>Gui</receiver>
   <slot>startStop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>Gui</receiver>
   <slot>open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionReload</sender>
   <signal>triggered()</signal>
   <receiver>Gui</receiver>
   <slot>reload()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionReloadSeed</sender>
   <signal>triggered()</signal>
   <receiver>Gui</receiver>
   <slot>reloadSeed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionStep</sender>
   <signal>triggered()</signal>
   <receiver>Gui</receiver>
   <slot>stepToTime()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionHelp</sender>
   <signal>triggered()</signal>
   <receiver>Gui</receiver>
   <slot>help()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>Gui</receiver>
   <slot>about()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionScreenshot</sender>
   <signal>triggered()</signal>
   <receiver>Gui</receiver>
   <slot>snapshot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoomIn</sender>
   <signal>triggered()</signal>
   <receiver>Gui</receiver>
   <slot>zoomIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoomOut</sender>
   <signal>triggered()</signal>
   <receiver>Gui</receiver>
   <slot>zoomOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCenter</sender>
   <signal>triggered()</signal>
   <receiver>Gui</receiver>
   <slot>centerCamera()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAutoZoom</sender>
   <signal>toggled(bool)</signal>
   <receiver>Gui</receiver>
   <slot>toggleAutoZoom(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action10steps</sender>
   <signal>triggered()</signal>
   <receiver>Gui</receiver>
   <slot>step()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action100steps</sender>
   <signal>triggered()</signal>
   <receiver>Gui</receiver>
   <slot>step()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action1step</sender>
   <signal>triggered()</signal>
   <receiver>Gui</receiver>
   <slot>step()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShowPlots</sender>
   <signal>toggled(bool)</signal>
   <receiver>Gui</receiver>
   <slot>toggleShowPlots(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startStop()</slot>
  <slot>open()</slot>
  <slot>reload()</slot>
  <slot>reloadSeed()</slot>
  <slot>step()</slot>
  <slot>step1()</slot>
  <slot>step10()</slot>
  <slot>step100()</slot>
  <slot>help()</slot>
  <slot>about()</slot>
  <slot>snapshot()</slot>
  <slot>zoomIn()</slot>
  <slot>zoomOut()</slot>
  <slot>toggleAutoZoom(bool)</slot>
  <slot>centerCamera()</slot>
  <slot>stepToTime()</slot>
  <slot>toggleShowPlots(bool)</slot>
 </slots>
</ui>
